ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/domain.scala

```diff
@@ -275,7 +274,8 @@ object domain {
       submitter: Ref.Party,
       ledgerEffectiveTime: Instant,
       maximumRecordTime: Instant,
-      ttl: Option[FiniteDuration],
+      submittedAt: Instant,
+      ttl: Instant,
```

Comment:
nitpick: should we stick to the expanded form as with other time related fields ("timeToLive")?

Reply:
I'd support that as well for improved readability. Also, to have consistent naming convention all `Duration`s vs. `Instant`s should be named consistently. I.e., this may be `maxTtl` if you wanted to follow the patterns at other places in this PR.